src/Craftify.Geometry.Domain/DoubleExtensions.cs

```diff
@@ -16,4 +16,6 @@ public static bool AlmostEqualTo(
     {
         return to.SubtractFrom(from).ToAbsolute() < tolerance;
     }
+
+    public static bool IsCloseToZero(this double value) => Math.Abs(value) < Defaults.Tolerance;
```

Comment:
Instead of hardcoding the tolerance here, can we pass it as a parameter and set a default value?
```csharp
public static bool IsCloseToZero(this double value, double tolerance = Defaults.Tolerance) => Math.Abs(value) < tolerance;
```